### DISCUSSION TITLE
Verticle Progress Bar Minimum Height

### DISCUSSION THREAD

**Discussion by CMurtagh-LGTM** [JakeStanger/ironbar#320] [Q&A]
Hi,
I've been trying to decrease height of a progress bar to the height of my bar.
Using the inspector I made sure that all the css had the `min-height` property set to 0.
Changing the length property changes the width.

Does any of you know what I am doing wrong?

`ironbar 0.14.0-pre`

**Comment by A-Cloud-Ninja** [comment]
If you post the CSS block in question (I'm assuming you mean `.popup-music .progress`) I can try to give you some help, I've styled my progress bar pretty heavily.

**Reply by A-Cloud-Ninja** [reply]
Oh sorry I completely forgot ironbar custom modules can have a progress bar. If you post your config + css I can poke at it and see if I can figure it out!

**Reply by CMurtagh-LGTM** [reply]
Here is what I've got currently 
https://github.com/CMurtagh-LGTM/dots/tree/master/ironbar
